scripts/rawadimover_integration_test.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This is an integration test for the RawADIMoverCronApp. This test
+# must be run by hand in a local development environment.
+#
+# To run it, do:
```

Comment:
You also need to add `crontabber.jobs=socorro.cron.crontabber_app.STAGE_NEW_JOBS` to `my.env` so that crontabber can find the job.

Reply:
Oh noes! I forgot this step. Sorry about that! I'll fix this now.